utils/CMakeLists.txt

```diff
@@ -2,12 +2,33 @@ if(ENABLE_OSCAP_UTIL)
 	file(GLOB UTILS_SOURCES "*.c")
 	add_executable(oscap ${UTILS_SOURCES})
 	target_link_libraries(oscap openscap)
+
```

Comment:
Would it make sense to have one `if(WIN32) ... else ... endif` block and put as many directives there, instead of repeating `if ... endif` or `if not ... endif`?

Reply:
Yes!